string-calculator.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="assets/styles/string-calculator.css">
+    <title>Title</title>
+</head>
+<body>
+<main class="calculator">
+    <section id="window" class="calc-window"></section>
```

Comment:
_UI improvements_: there is no possibility to enter the formula to some input and calculate it. For example, if I want to calculate the string "3*10" and just paste it to input.

Reply:
fixed